app/src/lib/ssh/ssh.ts

```diff
@@ -73,38 +71,3 @@ export async function getSSHEnvironment() {
 
   return baseEnv
 }
-
-const appName = __DEV__ ? 'GitHub Desktop Dev' : 'GitHub Desktop'
-const SSHKeyPassphraseTokenStoreKey = `${appName} - SSH key passphrases`
-
-async function getHashForSSHKey(keyPath: string) {
-  return getFileHash(keyPath, 'sha256')
-}
-
-/** Retrieves the passphrase for the SSH key in the given path. */
-export async function getSSHKeyPassphrase(keyPath: string) {
-  try {
-    const fileHash = await getHashForSSHKey(keyPath)
-    return TokenStore.getItem(SSHKeyPassphraseTokenStoreKey, fileHash)
-  } catch (e) {
-    log.error('Could not retrieve passphrase for SSH key:', e)
-    return null
-  }
-}
-
-/** Stores the passphrase for the SSH key in the given path. */
-export async function storeSSHKeyPassphrase(
-  keyPath: string,
-  passphrase: string
-) {
-  try {
-    const fileHash = await getHashForSSHKey(keyPath)
-    await TokenStore.setItem(
-      SSHKeyPassphraseTokenStoreKey,
-      fileHash,
-      passphrase
-    )
-  } catch (e) {
-    log.error('Could not store passphrase for SSH key:', e)
-  }
-}
```

Comment:
Just stuff I moved to `ssh-key-passphrase.ts` (except for `storeSSHKeyPassphrase`, which was removed).